### ISSUE TITLE
Feedback for assignment #2

### ISSUE THREAD

**Issue by barryross** [erickv/homework-landingpage#1]
# FEWD Week #2 Project: Styling with CSS

---

### Description

A hot, new NY Tech Startup, Relaxr, has approached you to help them develop a new landing page for their company. They've handed you the design file for the site, along with the copy and assets. You need to take these files and turn it into a landing page.

Student: Erick Vogel

Project URL: https://github.com/erickv/homework-landingpage__

|                                          |                                |                        |                          |
| ---------------------------------------- | ------------------------------ | ---------------------- | ------------------------ |
| Technical Requirements                   | Does Not Meet Expectations (0) | Meets Expectations (1) | Exceeds Expectations (2) |
| Use HTML5 structural elements (nav, header, footer) |                                |                        | x                        |
| Use a CSS Reset file in addition to your style.css file to style the page |                                |                        | x                        |
| Use IDs and Classes to to select and style elements on the page |                                |                        | x                        |
| Style your text with the Google Fonts provided by your style guide |                                | x                      |                          |
| Follow naming conventions, maintain consistency across .html and .css files and use best practices for naming IDs and Classes |                                |                        | x                        |
| Indent nested elements to increase your code's readability |                                |                        | x                        |
| TOTAL: 11/12                             |                                |                        |                          |

**Comment by barryross** [comment]
Hey Erick, great work with this assignment !  Excellent job hitting the tech requirements.  I left off one mark for the design spec just because there were a couple places where I didn't see the styling (for example, the `300` font weight for "the perfect personal...").  I also noticed many `font-sizes` were doubling those in the design spec.  I assume this was intentional though?  

Overall, great job and keep up the great work! 👍

**Comment by erickv** [comment]
Thank you, Barry, I had some problems adding the google fonts on the font-weight, the bold or semibold weren't working I tried the numbers and didn't work either. In regards of doubling the sizes, I used the one from the design specs and looked small. I'll be more careful next time.

**Comment by barryross** [comment]
Yup, you're welcome...thanks for letting me know.  Totally fine to adjust the font sizes if you would like them bigger, I would ask if you could please leave the original ones in the code, but commented out, just so I can see they match the design spec.  

That being said, the important part to me is that you are comfortable matching them, which it sounds like you are able to do with the font sizes.   

We can take a look at the font weight issue(s) tonight before class. 

Thanks!
